tidb-cloud/config-s3-and-gcs-access.md

```diff
@@ -9,7 +9,26 @@ If your source data is stored in Amazon S3 or Google Cloud Storage (GCS) buckets
 
 ## Configure Amazon S3 access
 
-To allow TiDB Cloud to access the source data in your Amazon S3 bucket, take the following steps to configure the bucket access for TiDB Cloud and get the Role-ARN.
+To allow TiDB Cloud to access the source data in your Amazon S3 bucket, you need to configure the bucket access for TiDB Cloud. You can use either of the following methods to configure the bucket access:
+
+- Use an Amazon access key: use the access key of an IAM user to access your Amazon S3 bucket.
```

Comment:
```suggestion
- Use an AWS access key: use the access key of an IAM user to access your Amazon S3 bucket.
```

Reply:
AWS is the product name. https://aws.amazon.com/premiumsupport/knowledge-center/create-access-key/